test/torch/tensors/test_additive_shared.py

```diff
@@ -474,6 +474,7 @@ def test_public_mul(workers, protocol):
     # assert (z == (t * y)).all()
 
 
+@pytest.mark.parametrize("protocol", ["snn", "fss"])
 def test_div(workers):
```

Comment:
You should do slightly more change:
```suggestion

@pytest.mark.parametrize("protocol", ["snn", "fss"])
def test_div(workers, protocol):
```
And then when you share, add the `protocol` keyword:
```
x = t.fix_prec(dtype="long").share(bob, alice, crypto_provider=james, protocol=protocol)
```
Do this for all the functions you have modified and it will work!

Reply:
👍